src/borg/archiver/create_cmd.py

```diff
@@ -90,7 +91,7 @@ def create_inner(archive, cache, fso):
                 paths_sep = eval_escapes(args.paths_delimiter) if args.paths_delimiter is not None else "\n"
                 if args.paths_from_command:
                     try:
-                        proc = subprocess.Popen(args.paths, stdout=subprocess.PIPE, preexec_fn=ignore_sigint)
+                        proc = subprocess.Popen(args.paths, stdout=subprocess.PIPE, preexec_fn=None if is_win32 else ignore_sigint)
```

Comment:
do we have some more of these in the tar related commands?

maybe do a global search for `subprocess.Popen`.